modules/nodes-cluster-limit-ranges-about.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/limit-ranges.adoc
+
+[id="nodes-cluster-limit-ranges-about_{context}"]
+= About limit ranges
+
+A limit range, defined by a LimitRange object, restricts resource 
+consumption in a project. In the project you can set specific resource 
+limits for a Pod, container, image, image stream, or 
+persistent volume claim (PVC).
+
+All requests to create and modify resources are evaluated against each
+LimitRange object in the project. If the resource violates any of the
+enumerated constraints, the resource is rejected. 
+
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+For {product-title} Pro, the maximum Pod memory is 3Gi. The minimum Pod or
+container memory that you can specify is 100Mi.
+
+For {product-title} Starter, the maximum Pod memory is 1Gi. The minimum Pod or
+container memory that you can specify is 200Mi.
+====
+endif::[]
+
+The following shows a limit range object for all components: Pod, container, 
+image, image stream, or PVC. You can configure limits for any or all of these 
+components in the same object. You create a different limit range object for 
+each project where you want to control resources.
+
+.Sample limit range object with all components defined
+
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "resource-limits"
+spec:
+  limits:
+    - type: "Pod"
+      max:
+        cpu: "2"
+        memory: "1Gi"
+      min:
+        cpu: "200m"
+        memory: "6Mi"
+    - type: "Container"
+      max:
+        cpu: "2"
+        memory: "1Gi"
+      min:
+        cpu: "100m"
+        memory: "4Mi"
+      default:
+        cpu: "300m"
+        memory: "200Mi"
+      defaultRequest:
+        cpu: "200m"
+        memory: "100Mi"
+      maxLimitRequestRatio:
+        cpu: "10"
+    - type: openshift.io/Image
+      max:
+        storage: 1Gi
+    - type: openshift.io/ImageStream
+      max:
+        openshift.io/image-tags: 20
+        openshift.io/images: 30
+    - type: "PersistentVolumeClaim"
+      max:
+      	storage: "2Gi"
```

Comment:
Alignment error.

Reply:
@zhouying7780 Thank you. There was a tab character there, In the text editor, the alignment looked correct. Here, it does not. Is this correct?
     max:
          storage: "2Gi"